spec/features/schools/add_appropriate_body_spec.rb

```diff
@@ -20,16 +20,20 @@
       given_there_is_a_school_and_an_induction_coordinator
       and_i_have_added_an_ect
       and_i_am_signed_in_as_an_induction_coordinator
```

Comment:
Set up the data still requires this methods as they are not part of the Page classes.